core/src/main/resources/hudson/model/AbstractBuild/tasks.jelly

```diff
@@ -27,7 +27,6 @@ THE SOFTWARE.
 -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout" xmlns:p="/lib/hudson/project">
-      <l:task contextMenu="false" href="${it.upUrl}" icon="icon-up icon-md" title="${%Back to Project}"/>
```

Comment:
Shouldn't the status be removed as well?

Reply:
The 'Status' link refers to the default child page for the project, and much like the updated Plugin Manager in #6783 all children pages should be there for easy navigation between them. I've updated the Design Library docs for this 👍

Reply:
I see. Then it makes sense to let it unchanged.